cpuid/Cargo.toml

```diff
@@ -5,4 +5,4 @@ authors = ["Amazon firecracker team <firecracker-devel@amazon.com>"]
 
 [dependencies]
 kvm = { path = "../kvm" }
-kvm_gen = { path = "../kvm_gen" }
+kvm_wrapper = { version = "0.1.0" }
```

Comment:
I don't want to enforce the version to be 0.1.0. I can use kvm_wrapper = "0.1.0". I will move the dependency so it can sit with its friends from crates.io.